internal/quaiapi/quai_api.go

```diff
@@ -676,6 +676,18 @@ func (s *PublicBlockChainQuaiAPI) UpdateDom(ctx context.Context, raw json.RawMes
 	s.b.UpdateDom(domUpdate.OldTerminus, domUpdate.NewTerminus, domUpdate.Location)
 }
 
+type RequestDomArgs struct {
```

Comment:
(nit) please name this more precisely. There could be many requests to dom

Reply:
renamed it to the function name to avoid confusion